CHANGELOG.md

```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v1.0.0](https://github.com/voxpupuli/puppet-puppet_certificate/tree/v1.0.0) (2023-05-19)
+## [v1.1.0](https://github.com/voxpupuli/puppet-puppet_certificate/tree/v1.1.0) (2024-02-01)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-puppet_certificate/compare/v1.0.0...v1.1.0)
+
+**Implemented enhancements:**
+
+- Add Puppet 8 support [\#17](https://github.com/voxpupuli/puppet-puppet_certificate/pull/17) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- New release to the forge [\#6](https://github.com/voxpupuli/puppet-puppet_certificate/issues/6)
+
+**Merged pull requests:**
+
+- README.md: Update code example [\#23](https://github.com/voxpupuli/puppet-puppet_certificate/pull/23) ([bastelfreak](https://github.com/bastelfreak))
+- Document PE workaround for auth.conf [\#19](https://github.com/voxpupuli/puppet-puppet_certificate/pull/19) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
enhancement? docs? :)

Reply:
I added the docs label but we don't handle that in the changelog. those are only documentational changes, not sure if those qualify as an enhancement.

Reply:
they kinda are, even if small, but I am fine either way